module/cpf.sh

```diff
@@ -26,17 +29,22 @@ declare -A COLORS=(
 )
 
 #. cpf:module -={
-function ::cpf:module_is_modified() {
+function ::cpf:module_is_unchanged() {
     local -i e=9
     local profile=$1
     local module=$2
-    cd ${SIMBOL_CORE?}
-    if [ -e "${SIMBOL_USER_MOD?}/${module}" ]; then
-        local path=$(readlink "${SIMBOL_USER_MOD?}/${module}")
-        local amended=$(:core:git status --porcelain "${path}"|wc -l)
-        [ ${PIPESTATUS[0]} -ne 0 ] || e=${amended}
+    local module_path
+
+    if [ "${profile}" = "core" ]; then
```

Comment:
I'm a bit worried about this, it's confusing since the "profile" and `SIMBOL_PROFILE` are what we use to determine the context under which simbol is operating; for example it would be set to `DEFAULT` by default, but then the user creates profiles for the context pertaining to their job.  So at AWS I have my profile set to `AWS`, and when I was at Sirca it was set to `SIRCA`.  The new term "core" is out-of-place here.